common/src/main/java/de/florianmichael/rclasses/common/access/DataTypeChecker.java

```diff
@@ -132,11 +201,44 @@ public static boolean isCharacter(final String input) {
      * @return True if the string is a UUID, false otherwise.
      */
     public static boolean isUUID(final String input) {
+        if (input.length() != 36) return false;
         try {
             UUID.fromString(input);
             return true;
         } catch (final IllegalArgumentException e) {
             return false;
         }
     }
+
+    /**
+     * Checks if the given string is a {@link BigInteger}
+     * @param input The string to check.
+     * @return True if the string is a BigInteger, false otherwise.
+     */
+    public static boolean isBigInteger(final String input) {
+        return isBigInteger(input, 10);
+    }
+
+    /**
+     * Checks if the given string is a {@link BigInteger}
+     * @param input The string to check.
+     * @param radix The radix used while parsing the input.
+     * @return True if the string is a BigInteger, false otherwise.
+     */
+    public static boolean isBigInteger(final String input, final int radix) {
+        if (input.isEmpty()) return false;
+        try {
+            new BigInteger(input, radix);
+            return true;
+        } catch (final NumberFormatException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Returns the size of the biggest possible string that can be used to encode a number within the given maximum value and radix.
+     */
+    private static int maximumStringSize(int radix, long maxValue) {
```

Comment:
I think we can actually make this method public / Maybe move it into StringUtils?